src/components/PageFooter.tsx

```diff
@@ -13,7 +13,7 @@ export interface PageFooterProps {
   privacyPolicy: string;
 }
 
-export const PageFooter = ({ title, location, socialMedia, privacyPolicy }: PageFooterProps) => {
+export const PageFooter = ({ title, location }: PageFooterProps) => {
```

Comment:
@NilsJacobsen Thanks! But we should offer social links somewhere haha

Reply:
That's true, it caused a big warning in the console. But yes we have to the the social links.